tests/unit/src/kakarot/instructions/test_system_operations.py

```diff
@@ -10,22 +10,35 @@
 from tests.utils.helpers import get_create2_address, get_create_address
 from tests.utils.uint256 import int_to_uint256
 
+ZERO_ACCOUNT = "0x0000000000000000000000000000000000000000"
+
 
 @pytest_asyncio.fixture(scope="module")
 async def system_operations(
-    starknet: Starknet, contract_account_class, account_proxy_class
+    starknet: Starknet, eth, contract_account_class, account_proxy_class
 ):
     return await starknet.deploy(
         source="./tests/unit/src/kakarot/instructions/test_system_operations.cairo",
         cairo_path=["src"],
         constructor_calldata=[
+            eth.contract_address,
             contract_account_class.class_hash,
             account_proxy_class.class_hash,
         ],
         disable_hint_validation=True,
     )
 
 
+@pytest_asyncio.fixture(scope="module")
+async def mint(system_operations, eth):
```

Comment:
this fixture is interesting; we could improve it by making it factory that would take as input an evm address + a value as int

```
@pytest_asyncio.fixture
def mint(eth, system_operations):
    async def _factory(evm_address, value):
        starket_contract_address = (
            await system_operations.compute_starknet_address(evm_address).call()
        ).result.contract_address
        await eth.mint(starket_contract_address, int_to_uint256(value)).execute()
```